app/formatColorCoding.js

```diff
@@ -0,0 +1,19 @@
+const MajorColorNames = [
+    "WHITE", "RED", "BLACK", "YELLOW", "VIOLET"
+];
+const MinorColorNames = [
+	"BLUE", "ORANGE", "GREEN", "BROWN", "SLATE"
+];
+function formatColorCoding(){
```

Comment:
I had difficulty finding this function, since it has no words in common with the requirement. When a developer wants to find this functionality, they would intuitively search for the words 'manual' or 'print'.
Try naming the functions by their purpose, not the implementation.